DataFormats/Portable/interface/PortableDeviceCollection.h

```diff
@@ -48,6 +48,28 @@ class PortableDeviceCollection {
     assert(reinterpret_cast<uintptr_t>(buffer_->data()) % Layout::alignment == 0);
   }
 
+  template <typename L = Layout>
+    requires requires { L::blocksNumber; }
+  PortableDeviceCollection(std::array<int32_t, L::blocksNumber> elements, TDev const& device)
```

Comment:
Is `elements` passed by value on purpose, or would it benefit from being a `const` reference?